src/controllers/controllermanager.h

```diff
@@ -85,7 +89,7 @@ class ControllerManager : public QObject {
     void stopPolling();
     void maybeStartOrStopPolling();
 
-    static QString presetFilenameFromName(QString name) {
+    static QString sanitizeString(QString name) {
```

Comment:
This function name is very generic and doesn't reveal what the function is supposed to do and for which use case it is supposed to be used. We need at least a comment or preferably a more descriptive name.

Reply:
I added a comment. If you have an idea for a good name let me know. The previous name was misleading because the was function is also used for other stuff, e.g. config keys.